create_test.go

```diff
@@ -40,9 +40,10 @@ func TestSequential(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	expected := fmt.Sprintf("%05v", 1)
```

Comment:
Are you sure this is correct?

Reply:
This means you're expecting _all_ filenames to be `00001`.

This test is testing that each file has the correct sequence id. Modifying a test like this to get your implementation to work means there is probably a bug with your implementation.

If I check out your PR, run:

```
go test -v -run=TestSequential -count=1
[..]
00001_add_indices.go
00001_add_users.go
00001_create_table.go
00001_update_users.go
```

This doesn't look right, I'm expecting them to be sequential:

```
00001
00002
00003
00004
```

Reply:
Ok. I fix this

Reply:
@mfridman done